docs/developer/decision-records/2022-07-29-self-description/README.md

```diff
@@ -0,0 +1,59 @@
+# Identity Hub - Self Description
+
+This document explains how the Self-Description of a dataspace participant will be stored into the Identity Hub and retrieved from it.
+
+## Context
+
+Gaia-X defines a set of rules that define the minimum baseline to be part of a Gaia-X Ecosystem. This set of rules is the so-called
+[Gaia-X Trust Framework](https://gaia-x.gitlab.io/policy-rules-committee/trust-framework/), which is centered around the _Self-Description_ document.
+
+Each dataspace participant (consumer, provider, federator) is represented by a Self-Description document.
+
+According to [spec](https://gaia-x.eu/wp-content/uploads/2022/06/Gaia-x-Architecture-Document-22.04-Release.pdf),
+"Self-Descriptions are W3C Verifiable Presentations in the JSON-LD format. Self-Description consist of a list of
+Verifiable Credentials. Verifiable Credentials themselves contain a list of Claims: assertions about Entities
+expressed in the RDF data model. Both Verifiable Credentials and Verifiable Presentations come with
+cryptographic signatures to increase the level of trust".
+
+Note that a Self-Description can aggregate Verifiable Credentials from different issuers, each having its own cryptographic signature.
+
+An example of valid Self-Description can be found [here](https://gitlab.com/gaia-x/gaia-x-community/gx-hackathon/gx-hackathon-4/-/blob/main/Example%20Self-Descriptions/participantA1digital.json), which corresponds to the one used during the Gaia-X Hackathon #4.
+
+Long term, the Verifiable Credentials from the Self-Description should be stored into the Identity Hub so that they can latter
```

Comment:
The reason to serve the self-description document via the IdentityHub ist to be able to dynamically generate the self-description at a later stage. If we serve the self-description already via the IdentityHub now, nothing changes for consumers accessing self-descriptions at a later stage when self-descriptions are generated.

We could add this information to the ADR.